client/blocks/login/two-factor-authentication/two-factor-actions.jsx

```diff
@@ -23,17 +21,14 @@ class TwoFactorActions extends Component {
 		isAuthenticatorSupported: PropTypes.bool,
 		isSmsSupported: PropTypes.bool,
 		twoFactorAuthType: PropTypes.string.isRequired,
```

Comment:
Not related to the changes but it would be great to add a prop type for `isPushSupported`, `sendSmsCode`, and `translate`.